### PR TITLE
Accounts and Alma 

### PR DESCRIPTION
**Pull request by kevinreiss**

### CODE REVIEW

File: accounts.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+** fpul.princeton.edu
+* Other Tools
+** Princeton and Slavery - craftcms
+
+
+# Other External Services
+
+* Ex Libris Developers Network (One of the Alma team add you to the "Princeton" Team)
+* Datadog - monitoring and alerts
+* Ciricle CI - Via Github Account
```
Comment (christinach) [inline accounts.md:50]: small typo, change to [CircleCi](https://circleci.com/)

File: accounts.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+* Other Tools
+** Princeton and Slavery - craftcms
+
+
+# Other External Services
+
+* Ex Libris Developers Network (One of the Alma team add you to the "Princeton" Team)
+* Datadog - monitoring and alerts
+* Ciricle CI - Via Github Account 
+*
```
Comment (christinach) [inline accounts.md:51]: [Honeybadger](https://www.honeybadger.io/)

File: accounts.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+** recap.princeton.edu
+** fpul.princeton.edu
+* Other Tools
+** Princeton and Slavery - craftcms
+
+
+# Other External Services
+
+* Ex Libris Developers Network (One of the Alma team add you to the "Princeton" Team)
+* Datadog - monitoring and alerts
```
Comment (christinach) [inline accounts.md:49]: [Datadog](https://www.datadoghq.com/)

File: accounts.md (hunk 2 above)
Comment (christinach) [reply accounts.md:51]: [Amazon AWS](https://princeton.edu/aws)

File: alma.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+# Alma is our Library Services Platform
+
+## Access
+
+### User Login
+* Production Login URL
```
Comment (christinach) [inline alma.md:6]: [Production Login URL](https://princeton.alma.exlibrisgroup.com/SAML)

File: alma.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+* Sandbox Login URL
```
Comment (christinach) [inline alma.md:7]: [Sandbox Login URL](https://princeton-psb.alma.exlibrisgroup.com/SAML)

Comment (christinach) [reply alma.md:7]: [Sensu](https://lib-monitor.princeton.edu/dashboard)

File: accounts.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+* Alma
+** Alma Prod 
+** Alma Sandbox
+* SCSB (ReCAP Middleware)
+** Prod
+** UAT
+* POD (Platform for Open Data)
+
+# Locally Hosted Catalog Related Applications
+* Aeon
```
Comment (carolyncole) [inline accounts.md:28]: May want to mention they are only accessible via remote desktop

File: accounts.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+All developers need a github profile handle with a public key to push and pull code to our two Princeton github communities. SSH Access to all servers is also through the use of this public key. 
+
+# Services attached to your Princeton SSO Account (i.e. DUO, Your University Account, the netid account)
+* University Email
+* VPN, aka Secure Remote Access (GlobalProtect Client Preferred)
+* Google Drive
+* Libanswers (Patron facing tickets) - Kevin can grant access
+* SN@P (Staff facing tickets) - Request access from Tracy Hall in Operations
+* Approvals (Library Travel Requests) - should be automatically added to this via
+  Peoplesoft
```
Comment (carolyncole) [inline accounts.md:15]: Can you push this up a line?  I thought it was something separate, since it was on it's own line

**Review by carolyncole — CHANGES_REQUESTED** [review]

File: alma.md (hunk 5 above)
Comment (christinach) [reply alma.md:7]: [Pulbot](https://github.com/pulibrary/pulbot) for deployments

**Review by sandbergja — APPROVED** [review]

**Pull request merged**